Makefile

```diff
@@ -166,6 +166,11 @@ build.exampleapps:
 	@examples/apps/helloworld/build-services.sh "$(APP_VER_ABBR)"
 	@examples/apps/bookinfo/build-services.sh "$(APP_VER_ABBR)"
 
+build.exampleappstesting: release.sidecar
+	@echo "--> building example apps for testing"
+	@examples/apps/helloworld/build-services.sh "$(APP_VER_ABBR)" "examples"
+	@examples/apps/bookinfo/build-services.sh "$(APP_VER_ABBR)" "examples"
+
```

Comment:
build.exampleapps is the standalone build for the apps.  So it will generate the yamls and it will generate the images with the download links.
build.exampleappstesting is for the automated testing path.  So it will build the release tars and generate the images using the local files.